src/test/kotlin/org/jitsi/rtp/rtcp/rtcpfb/transport_layer_fb/tcc/RtcpFbTccPacketTest.kt

```diff
@@ -136,21 +138,27 @@ class RtcpFbTccPacketTest : ShouldSpec() {
             "with RLE" {
                 val rtcpFbTccPacket = RtcpFbTccPacket(tccRleData.array(), tccRleData.arrayOffset(), tccRleData.limit())
                 should("parse the values correctly") {
-                    rtcpFbTccPacket.forEach { (seqNum, deltaTicks) ->
-                        expectedTccRlePacketInfo shouldContainKey seqNum
-                        withClue("seqNum $seqNum deltaTicks") {
-                            deltaTicks shouldBe expectedTccRlePacketInfo[seqNum]
+                    rtcpFbTccPacket.forEach {
+                        it should beInstanceOf<ReceivedPacketReport>()
+                        if (it is ReceivedPacketReport) {
```

Comment:
you can use `it as ReceivedPacketReport` to get a smart cast and avoid the `if` here (and below)